tests/suit/vsphere_test.go

```diff
@@ -12,7 +12,7 @@ import (
 
 const (
 	vsphereProviderName = "vsphere-provider"
-	vsphereStorageClass = "standard"
+	vsphereStorageClass = "nfs-csi"
```

Comment:
just to align with other suits? any other reason?

Reply:
that's also part of the 'debug' commit :)
but now that the test passes without the other commit - it seems that this is the reason for the failures of this test (the standard storage class)